examples/kotlin-kts/README.md

```diff
@@ -0,0 +1,32 @@
+# Kotlin-KTS Example
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/Kotlin/kotlinx-benchmark)
+
+## Project Structure
+
+Inside of this example, you'll see the following folders and files:
+
+```
+/
+├── build.gradle.kts
+└── main/
+    └── src/
+        └── KtsTestBenchmark.kt
+```
+
+## Tasks
+
+All tasks can be run from the root of the project, from a terminal:
+
+| Task Name | Action |
+| --- | --- |
+| `gradle assembleBenchmarks` | Generate and build all benchmarks in the project |
+| `gradle benchmark` | Execute all benchmarks in the project |
+| `gradle mainBenchmark` | Execute benchmark for 'benchmarks' |
```

Comment:
Should be 'main' in this example project.

Reply:
Thank you for your pointing this out and for your feedback.

Reply:
This comment doesn't seem to be resolved.
Should be something like `Runs benchmarks for 'main' benchmark target`.

Reply:
My apologies, I believe I resolved it and lost my changes without saving. Will address